Revit_Core_Engine/Modify/SetType.cs

```diff
@@ -132,7 +154,7 @@ public static bool ISetType(this Element element, IBHoMObject bHoMObject, RevitS
         /****              Private Methods              ****/
         /***************************************************/
 
-        public static bool TrySetTypeFromString(this Element element, IBHoMObject bHoMObject, RevitSettings settings)
+        private static bool TrySetTypeFromString(this Element element, IBHoMObject bHoMObject, RevitSettings settings)
```

Comment:
Can I suggest you add a description in this private class as well? Just for due diligence. Also check a couple of compliance warnings.

Reply:
Fair suggestion @tiagogrossi. Descriptions added!